docs/notebooks/Berger_POPC_25ns_buildH_calcOP.ipynb

```diff
@@ -0,0 +1,1342 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "OPs = df[\"OP\"].iloc[18:46]\n",
+    "\n",
+    "x = np.arange(len(OPs))\n",
+    "plt.scatter(x, -OPs)\n",
```

Comment:
Why a minus in "-OPs"?

Reply:
The reason is well explained on Samuli's blog : http://nmrlipids.blogspot.com/2014/11/new-version-of-manuscript.html?showComment=1416584781727#c8832795399216522099
Most OP are negative, and so far il literature people have looked to absolute values. Thus they are used to look at OP plots that way.

Reply:
oh yes indeed. I forgot about that :)